### DISCUSSION TITLE
A problem occured in Sumatra PDF 3.5.2

### DISCUSSION THREAD

**Discussion by brownbearbb** [sumatrapdfreader/sumatrapdf#4010] [Ideas]
I am a Total Commander user. I use the slister.wlx plugin. It is a marvelous plugin. It calls SumatraPDF.exe to quickly preview pdf files. I often use keyboard arrow keys to move the cursor to see previews of different pdf documents in the same folder one by one. But when I use Sumatra PDF version 3.5.2, I found that whenever I select a new pdf document, the selected file will lose focus and become unselected when the file preview is displayed. I had to use the mouse to select the next file I wanted to preview. This was annoying. So I switched back to Sumatra PDF 3.4.5 and the problem disappeared and I was able to use slister again as before.

I hope that the new version of Sumatra PDF will solve this problem, i.e., that when using slister to quickly preview a pdf, the selected files will remain selected. Thank you very much!

**Comment by GitHubRulesOK** [comment]
sLister has been using different variants of SumatraPDF since at least 2007 ? but WLX on its own rarely has issues as it only is driven by extensions. here is that old Lister with sLister update using current pre-release 17386. 

At times there have been conflicts between TC use of Hot Keys and SumatraPDF use of hot keys and usually those are resolved in the WLX file so I am using 2025 WLX 1.20 not older sLister 1.9 from 2011 

There has been much discussion about <kbd>4</kbd><kbd>n</kbd><kbd>p</kbd> and in SumatraPDF with TC those should have now been resolved.

<img width="985" height="667" alt="image" src="https://github.com/user-attachments/assets/bcfac298-c7db-40e4-918d-30ffdb4bcb08" />
